utils/mgmt_system/google.py

```diff
@@ -0,0 +1,269 @@
+# coding: utf-8
+"""Backend management system classes
+
+Used to communicate with providers without using CFME facilities
+"""
+
+import time
+from oauth2client.client import flow_from_clientsecrets
+from oauth2client.file import Storage
+from oauth2client.tools import run
+from googleapiclient.discovery import build
+from utils.mgmt_system.base import MgmtSystemAPIBase, VMInfo
+from utils.mgmt_system.exceptions import VMInstanceNotFound
+from utils.log import logger
+from utils.wait import wait_for
+
+
+class GoogleCloudSystem (MgmtSystemAPIBase):
+    """
+    Client to Google Cloud Platform API
+
+    """
+
+    states = {
+        'running': ('RUNNING',),
+        'stopped': ('TERMINATED',),
+        'starting': ('STAGING'),
+        'stopping': ('STOPPING'),
+    }
+
+    def __init__(self, **kwards):
```

Comment:
We would usually name this kwargs, keyword arguments, probably nice to change it to keep consistency